<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPlayer Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Video:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="videoComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Audio:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="audioComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="autoSyncCheckBox">
       <property name="text">
        <string>Audio/video auto synchronization</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Synchronization factor:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="syncFactorSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Extra options:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="cmdOptionsLineEdit">
       <property name="toolTip">
        <string>Extra command line options</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoSyncCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>syncFactorSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
